snap/snapcraft.yaml

```
@@ -2,7 +2,9 @@ name: edgex-device-rest
 base: core20
 type: app
 license: Apache-2.0
-adopt-info: metadata
+summary: EdgeX Device REST Service
```

This isn't the codename but the summary. A better English summary is:
```suggestion
summary: EdgeX REST Device Service
```